src/benchmarklib/benchmark_runner.cpp

```diff
@@ -271,7 +271,7 @@ void BenchmarkRunner::_benchmark_individual_queries() {
     const auto duration_seconds = static_cast<float>(result.duration_ns) / 1'000'000'000;
     const auto items_per_second = static_cast<float>(result.num_iterations) / duration_seconds;
 
-    std::cout << "  -> Executed " << result.num_iterations << " times in " << duration_seconds << " seconds ("
+    std::cout << "  -> Executed " << result.num_iterations.load() << " times in " << duration_seconds << " seconds ("
```

Comment:
Why did this become necessary?

Reply:
There is no `operator<<(std::ostream&, std::atomic<int>)`, so this previously used `operator<<(..., int)` via [implicit conversion](https://en.cppreference.com/w/cpp/atomic/atomic/operator_T). With the introduction of operator<< overloads for STRONG_TYPEDEFs, there are multiple candidate `operator<<` now - those for the primitive data types and those for the strong typedefs. Calling load() removes the ambiguity.